Demo/SnapshotTests/BroadcastMessageTests.swift

```diff
@@ -8,7 +8,7 @@ import XCTest
 class BroadcastMessageTests: XCTestCase {
     func testBroadcastModelCanReplaceHTMLLinkWithAttributedStringWithLinkAttrbutes() {
         // Given
-        let linkURL = URL(string: "https://www.finn.no/")!
+        let linkURL = URL(string: "https://www.finn.no/")! // swiftlint:disable:this force_unwrapping
```

Comment:
Instead of special-casing small things like this in the demo target, how about using a nested child config which allows more? Ref. last section in docs: https://realm.github.io/SwiftLint/index.html

Reply:
I just special cased it for quick fixing now. I think we could quite easily avoid most of the force unwraps in Demo as well, so I hope we will get less of them in the future by having it on 🙂